lib/toml/multilined.rb

```diff
@@ -0,0 +1,11 @@
+# Used in primitive.citrus
+module TomlMultilineString
+  def value
+    aux = captures[:text].first.value
+
+    # Remove spaces on multilined Singleline strings
+    aux.gsub!(/\\\r?\n[\n\t\r ]*/, '')
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.